spec/index.html

```diff
@@ -8765,9 +8769,9 @@ <h5>Grouping and Aggregation</h5>
 Let E := [], a list of pairs of the form (variable, expression)
 
 If Q contains GROUP BY exprlist
-   Let G := Group(exprlist, P)
-Else If Q contains an aggregate in SELECT, HAVING, ORDER BY
-   Let G := Group((1), P)
+   Let G := Group(exprlist, ToList(P))
+Else If Q contains an aggregate in <code>SELECT</code>, <code>HAVING</code>, <code>ORDER BY</code>
+   Let G := Group((1), ToList(P))
```

Comment:
It does not seems to cover the "in case of implicit grouping the single group must exist and might be empty" behavior. But it's not a problem for this MR, just noticing this here.

Reply:
@Tpt Good catch! However, I think that avoiding to use the Group operator is not possible because the translation algorithm does not know whether `P` produces an empty result. So, instead, the definition of the Group operator must be extended to handle this edge case differently.

Notice also that this problem is independent of the changes in this PR; it is also exists in the current spec text. To avoid overloading the PR, to you mind creating a separate issue?

Reply:
Thank you! I have opened #105.